packages/eslint-config-react-app/README.md

```diff
@@ -19,7 +19,7 @@ If you want to use this ESLint configuration in a project not built with Create
 First, install this package, ESLint and the necessary plugins.
 
 ```sh
-npm install --save-dev eslint-config-react-app @typescript-eslint/eslint-plugin@3.x @typescript-eslint/parser@3.x babel-eslint@10.x eslint@7.x eslint-plugin-flowtype@5.x eslint-plugin-import@2.x eslint-plugin-jsx-a11y@6.x eslint-plugin-react@7.x eslint-plugin-react-hooks@4.x
+npm install --save-dev eslint-config-react-app @typescript-eslint/eslint-plugin@3.x @typescript-eslint/parser@3.x babel-eslint@10.x eslint@7.x eslint-plugin-flowtype@5.2.0 eslint-plugin-import@2.22.0 eslint-plugin-jsx-a11y@6.3.1 eslint-plugin-react@7.20.3 eslint-plugin-react-hooks@4.0.8
```

Comment:
Don't forget carrots, otherwise specific versions are installed

```suggestion
npm install --save-dev eslint-config-react-app @typescript-eslint/eslint-plugin@^3.0.0 @typescript-eslint/parser@^3.0.0 babel-eslint@^10.0.0 eslint@^7.0.0 eslint-plugin-flowtype@^5.2.0 eslint-plugin-import@^2.22.0 eslint-plugin-jsx-a11y@^6.3.1 eslint-plugin-react@^7.20.3 eslint-plugin-react-hooks@^4.0.8
```